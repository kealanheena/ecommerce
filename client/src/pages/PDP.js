import React from 'react';

function PDP() {
  return (
    <div className="pdpPage">
      <h2>pdp page</h2>
    </div>
  );
}

export default PDP;
