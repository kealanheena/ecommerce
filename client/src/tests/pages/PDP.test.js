import React from 'react';
import { render } from '@testing-library/react';
import PDP from '../../pages/PLP';

test('matches snapshot', () => {
  const renderedPDP = render(<PDP />);
  expect(renderedPDP).toMatchSnapshot();
});
