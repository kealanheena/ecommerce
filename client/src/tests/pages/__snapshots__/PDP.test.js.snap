// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="plpPage container"
      >
        <div
          class="row"
        >
          <h2>
            PLP page
          </h2>
        </div>
        <div
          class="row"
        >
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Incredible Granite Towels"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Incredible Granite Towels
              </h5>
              <p>
                £394.00
              </p>
              <p>
                Practical, Generic, Handmade
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Tasty Rubber Ball"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Tasty Rubber Ball
              </h5>
              <p>
                £856.00
              </p>
              <p>
                Incredible, Practical, Gorgeous
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Fantastic Cotton Computer"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Fantastic Cotton Computer
              </h5>
              <p>
                £909.00
              </p>
              <p>
                Intelligent, Gorgeous, Gorgeous
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Gorgeous Soft Ball"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Gorgeous Soft Ball
              </h5>
              <p>
                £812.00
              </p>
              <p>
                Practical, Sleek, Gorgeous
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Tasty Plastic Table"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Tasty Plastic Table
              </h5>
              <p>
                £127.00
              </p>
              <p>
                Refined, Handmade, Sleek
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Refined Plastic Mouse"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Refined Plastic Mouse
              </h5>
              <p>
                £944.00
              </p>
              <p>
                Ergonomic, Rustic, Handcrafted
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Licensed Soft Pants"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Licensed Soft Pants
              </h5>
              <p>
                £376.00
              </p>
              <p>
                Ergonomic, Licensed, Awesome
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Gorgeous Cotton Chair"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Gorgeous Cotton Chair
              </h5>
              <p>
                £697.00
              </p>
              <p>
                Licensed, Rustic, Handcrafted
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Licensed Steel Tuna"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Licensed Steel Tuna
              </h5>
              <p>
                £596.00
              </p>
              <p>
                Ergonomic, Incredible, Gorgeous
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Handcrafted Frozen Ball"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Handcrafted Frozen Ball
              </h5>
              <p>
                £828.00
              </p>
              <p>
                Fantastic, Generic, Rustic
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Rustic Soft Soap"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Rustic Soft Soap
              </h5>
              <p>
                £16.00
              </p>
              <p>
                Incredible, Sleek, Handmade
              </p>
            </div>
          </div>
          <div
            class="col-4"
          >
            <div
              class="product-card"
            >
              <img
                alt="Handmade Metal Car"
                src="http://placehold.jp/640x480.png?text=placeholder"
                width="100%"
              />
              <h5>
                Handmade Metal Car
              </h5>
              <p>
                £944.00
              </p>
              <p>
                Small, Unbranded, Licensed
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="plpPage container"
    >
      <div
        class="row"
      >
        <h2>
          PLP page
        </h2>
      </div>
      <div
        class="row"
      >
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Incredible Granite Towels"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Incredible Granite Towels
            </h5>
            <p>
              £394.00
            </p>
            <p>
              Practical, Generic, Handmade
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Tasty Rubber Ball"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Tasty Rubber Ball
            </h5>
            <p>
              £856.00
            </p>
            <p>
              Incredible, Practical, Gorgeous
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Fantastic Cotton Computer"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Fantastic Cotton Computer
            </h5>
            <p>
              £909.00
            </p>
            <p>
              Intelligent, Gorgeous, Gorgeous
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Gorgeous Soft Ball"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Gorgeous Soft Ball
            </h5>
            <p>
              £812.00
            </p>
            <p>
              Practical, Sleek, Gorgeous
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Tasty Plastic Table"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Tasty Plastic Table
            </h5>
            <p>
              £127.00
            </p>
            <p>
              Refined, Handmade, Sleek
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Refined Plastic Mouse"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Refined Plastic Mouse
            </h5>
            <p>
              £944.00
            </p>
            <p>
              Ergonomic, Rustic, Handcrafted
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Licensed Soft Pants"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Licensed Soft Pants
            </h5>
            <p>
              £376.00
            </p>
            <p>
              Ergonomic, Licensed, Awesome
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Gorgeous Cotton Chair"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Gorgeous Cotton Chair
            </h5>
            <p>
              £697.00
            </p>
            <p>
              Licensed, Rustic, Handcrafted
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Licensed Steel Tuna"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Licensed Steel Tuna
            </h5>
            <p>
              £596.00
            </p>
            <p>
              Ergonomic, Incredible, Gorgeous
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Handcrafted Frozen Ball"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Handcrafted Frozen Ball
            </h5>
            <p>
              £828.00
            </p>
            <p>
              Fantastic, Generic, Rustic
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Rustic Soft Soap"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Rustic Soft Soap
            </h5>
            <p>
              £16.00
            </p>
            <p>
              Incredible, Sleek, Handmade
            </p>
          </div>
        </div>
        <div
          class="col-4"
        >
          <div
            class="product-card"
          >
            <img
              alt="Handmade Metal Car"
              src="http://placehold.jp/640x480.png?text=placeholder"
              width="100%"
            />
            <h5>
              Handmade Metal Car
            </h5>
            <p>
              £944.00
            </p>
            <p>
              Small, Unbranded, Licensed
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
